sdk/eventhub/event-hubs/src/receiver.ts

```diff
@@ -22,16 +22,24 @@ export interface EventIteratorOptions {
    */
   // prefetchCount?: number;
   /**
-   * Signal to cancel the operation.
+   * An `AbortSignal` instance to signal the `EventIterator` to cancel the operation.
    */
   abortSignal?: AbortSignalLike;
 }
 
 /**
- * The EventHubConsumer class can be used to receive messages in a batch or by registering handlers.
+ * A consumer is responsible for reading `EventData` from a specific Event Hub partition
+ * and as a member of a specific consumer group.
+ *
+ * A consumer may be exclusive, which asserts ownership over the partition for the consumer
+ * group to ensure that only one consumer from that group is reading the from the partition.
+ *
+ * A consumer may also be non-exclusive, allowing multiple consumers from the same consumer
+ * group to be actively reading events from the partition.
```

Comment:
I don't see the benefits of talking about exclusive, non-exclusiveness here. When creating the consumer/producer, yes. But not on a class that we don't expect users to instantiate directly